src/commoncode/fileutils.py

```diff
@@ -540,7 +540,7 @@ def copytime(src, dst):
     if hasattr(os, 'utime'):
         try:
             os.utime(dst, (st.st_atime, st.st_mtime))
-        except OSError, why:
+        except OSError as why:
```

Comment:
Since, one exception has left in #1457 .

Reply:
:+1: